cookbook/symfony1.rst

```diff
@@ -200,14 +200,14 @@ Applications
 ------------
 
 In a symfony1 project, it is common to have several applications: one for the
-frontend and one for the backend for instance.
+front-end and one for the back-end for instance.
```

Comment:
I'm ok with anything - `frontend` looks best to me, but it seems to be the least correct (I was surprised to see)